src/library/scala/collection/View.scala

```diff
@@ -89,34 +92,39 @@ object View extends IterableFactory[View] {
         def hasNext: Boolean = notConsumed
       }
     override def knownSize: Int = 1
+    override def isEmpty: Boolean = false
```

Comment:
Shouldn't all of these classes be `final`?

Reply:
I'm not familiar enough with this part of the new collections to know.

Reply:
Ping @szeiger

Reply:
See my comment on the other PR: https://github.com/scala/scala/pull/6799/files#r198779686